update.go

```diff
@@ -41,6 +43,9 @@ func updateCheck(ctx *ntcontext) {
 }
 
```

Comment:
(Github won't let me comment on a line outside the diff...)

https://github.com/lawl/NoiseTorch/blob/c6de9619d668f72f284b6b6a257f0d6eac9f88d6/update.go#L103

This is not true anymore. A `panic()` could be recovered. I think if the public key decoding fails we should just err on the side of caution and `os.Exit(1)` to ensure no further update code is ran under any circumstance.

Reply:
Should probably log it too.